misc/services/camel/de-metas-camel-externalsystems/core/src/test/resources/de/metas/camel/externalsystems/core/to_mf/0_LogMessageRequest.json

```diff
@@ -4,5 +4,5 @@
     "tableName": "API_Request_Audit",
     "recordId": 100001
   },
-  "logMessage": "Error: JsonErrorItem(message=HTTP operation failed invoking www.does-not-matter.com with statusCode: 500, redirectLocation: Location, detail=null, stackTrace=org.apache.camel.http.base.HttpOperationFailedException: HTTP operation failed invoking www.does-not-matter.com with statusCode: 500, redirectLocation: Location, parameters={}, adIssueId=null, orgCode=null, sourceClassName=null, sourceMethodName=null, issueCategory=null, throwable=null)"
+  "logMessage": " Error: JsonErrorItem(message=HTTP operation failed invoking www.does-not-matter.com with statusCode: 500, redirectLocation: Location, detail=null, stackTrace=org.apache.camel.http.base.HttpOperationFailedException: HTTP operation failed invoking www.does-not-matter.com with statusCode: 500, redirectLocation: Location, parameters={}, adIssueId=null, orgCode=null, sourceClassName=null, sourceMethodName=null, issueCategory=null, throwable=null)"
```

Comment:
is there a story around this heading space, which i should know about 🤔 ?

Reply:
@metas-ts if you check de.metas.camel.externalsystems.core.to_mf.ErrorReportRouteBuilder#prepareErrorLogMessage, you will see that the error message is built with that space in front. Now, I'm not sure why that's the case, but I updated the json file to include the tableName and recordId and I might have removed that space by mistake. And the test was failing because of that.

Reply:
not really, it was just left behind after removing the api audit request id from the message.. will remove it, thx!